snap/snapcraft.yaml

```diff
@@ -73,9 +79,10 @@ parts:
       ./usr/share/srsran/sib.conf.example: ./usr/share/srsran/sib.conf
       ./usr/share/srsran/user_db.csv.example: ./usr/share/srsran/user_db.csv
       ./usr/share/srsran/rb.conf.example: ./usr/share/srsran/rb.conf
-    override-build: |
-      set -x
-      pushd $SNAPCRAFT_PART_SRC
-      sed -ie 's/native/x86_64/g' CMakeLists.txt
-      popd
-      snapcraftctl build
+    override-prime: |
+      snapcraftctl prime
+      sed -i '/device_args = tx_port=tcp/s/^#//g' ./usr/share/srsran/ue.conf
```

Comment:
I'm not a huge fan of those `sed` command as they require us to assume the content of the config file. I'm fine if we have the complete config files here.

Reply:
Yes, I also thought of this, but we are using a specific release tag, not the latest version of the code.
So maybe we can assume it won't change.
I'm happy to take it out to the README instead, let me know what you think :)

Reply:
This assumption is somewhat applicable to both the `sed` and the complete config file. It's more that f we provide the config file we **know** what we are publishing. Here we were doing a `sed` on a file that we don't know the content of.

Reply:
Good point. Done.